src/gui/clips/SampleClipView.cpp

```diff
@@ -268,15 +277,29 @@ void SampleClipView::paintEvent( QPaintEvent * pe )
 	float den = Engine::getSong()->getTimeSigModel().getDenominator();
 	float ticksPerBar = DefaultTicksPerBar * nom / den;
 
-	float offset =  m_clip->startTimeOffset() / ticksPerBar * pixelsPerBar();
-	QRect r = QRect( offset, spacing,
-			qMax( static_cast<int>( m_clip->sampleLength() * ppb / ticksPerBar ), 1 ), rect().bottom() - 2 * spacing );
+	// Visualize
```

Comment:
Maybe I'm picky, but I just find comments like this so redundant :shrug: I mean I would be concerned if we weren't doing something related to visualizing here anyways lol